### PR TITLE
当选择的时候更改this.lastInputValue.

### PR DESCRIPTION
**Pull request by percyi**
当选择的时候更改this.lastInputValue.

### CODE REVIEW

**Comment by paranoidjk** [comment]
https://github.com/react-component/select/issues/184 ci 点进去看，lint 没过
```bash
> rc-select@6.8.2 lint /home/travis/build/react-component/select
> rc-tools run lint
rc-tools run lint
/home/travis/build/react-component/select/src/DropdownMenu.jsx
  69:33  warning  Missing function expression name           func-names
  69:41  error    Missing space before function parentheses  space-before-function-paren
  69:47  error    Missing space before opening brace         space-before-blocks
  72:24  warning  Missing function expression name           func-names
  72:32  error    Missing space before function parentheses  space-before-function-paren
  72:38  error    Missing space before opening brace         space-before-blocks
  73:28  error    Trailing spaces not allowed                no-trailing-spaces
  75:8   error    Missing semicolon                          semi
/home/travis/build/react-component/select/src/Select.jsx
  32:10  warning  Missing function expression name  func-names
/home/travis/build/react-component/select/tests/__mocks__/rc-trigger.js
  7:28  warning  Missing function expression name  func-names
/home/travis/build/react-component/select/tests/FilterMixin.spec.js
  101:14  warning  Unexpected console statement  no-console
  102:14  warning  Unexpected console statement  no-console
✖ 12 problems (6 errors, 6 warnings)

```

**Comment by percyi** [comment]
test那边过不去了

**Comment by paranoidjk** [comment]
`npm run test -- -u`

**Comment by percyi** [comment]
PASS  tests/DropdownMenu.spec.js
  ● Console
    console.error node_modules/fbjs/lib/warning.js:36
      Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
    console.error node_modules/fbjs/lib/warning.js:36
      Warning: DropdownMenu: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.
 FAIL  tests/Select.combobox.spec.js
  ● Console
    console.error node_modules/fbjs/lib/warning.js:36
      Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
    console.error node_modules/fbjs/lib/warning.js:36
      Warning: DropdownMenu: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.

File: src/DropdownMenu.jsx [hunk 1]
```diff
@@ -66,11 +66,18 @@ const DropdownMenu = React.createClass({
     } = props;
     if (menuItems && menuItems.length) {
       const menuProps = {};
+      const setLastInputValue = function setLastInputValue(text) {
```
Comment (paranoidjk) [inline src/DropdownMenu.jsx]: use arrow function

**Comment by paranoidjk** [comment]
ignore those warning, or PR to fix React15.5 warning is welcome

**Comment by percyi** [comment]
how to ignore those warnings?

**Comment by yesmeck** [comment]
没重现你在 #181 里说的问题，PR 先关了，如果还有问题的话，最好附 demo 说明下。

**Pull request closed without merging**